src/nfs.c

```diff
@@ -596,6 +612,34 @@ static int nfs_read(void) {
 }
   /* #endif KERNEL_LINUX */
 
+#elif KERNEL_NETBSD
+static int nfs_read(void) {
+  struct nfsstats ns;
+  size_t size = sizeof(ns);
+  int mib[] = {CTL_VFS, 2, NFS_NFSSTATS};
```

Comment:
Might add a comment about the meaning of 2.

Reply:
That is documented in line 623
```
  /* NetBSD reports v2 statistics mapped to v3 and doen't yet support v4 */
```